docs/src/configuration/app/workers.md

```diff
@@ -18,10 +18,10 @@ workers:
         size: S
         commands:
             start: |
-                php worker.php
+                ./worker
 ```
 
-That defines a single worker named `queue`, which will be a "small" container, and wil run the command `php worker.php` on startup.  If `worker.php` ever exits it will be automatically restarted.
+That defines a single worker named `queue`, which will be a "small" container, and wil run the command `./worker` on startup.  If `worker` ever exits it will be automatically restarted.
```

Comment:
```suggestion
That defines a single worker named `queue`, which will be a "small" container, and will run the command `./worker.sh` on startup.  If `worker` ever exits it will be automatically restarted.
```